modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-relationships/account-to-partner-level.json

```diff
@@ -0,0 +1,11 @@
+{
+	"deletionType": "cascade",
+	"label": {
+		"en-US": "Partner Level to Account"
```

Comment:
account-to-partner-level or partner-level-to-account?

Reply:
partner-level-to-account